client/src/components/UI/Toast.tsx

```diff
@@ -0,0 +1,58 @@
+import React,{useCallback, useEffect} from "react";
+import { toastActions } from "../../store/toast";
+import { useDispatch } from "react-redux";
+import styles from "./Toast.module.css";
+
+export interface IToastProps {
+    id: number;
+    title: string;
+    description: string;
+    backgroundColor: string;
+}
+
+type toastProps = {
+    toastList: IToastProps[];
+    position: string;
+}
+
+const Toast : React.FC<toastProps> = ({ toastList, position }) => {
+
+  const dispatch = useDispatch();
+  const deleteToast = useCallback((id:number) => {
+      const toastListItem = toastList.filter((e) => e.id !== id);
+      dispatch(toastActions.removeToast(toastListItem));
+    },
+    [toastList,dispatch]
+  );
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      if (toastList.length) {
+        deleteToast(toastList[0].id);
+      }
+    }, 3000);
```

Comment:
the timeout should also be configurable,. so the user can also pass the interval for each toast,. if not passed then it should use a default value